packages/react-instantsearch-hooks-dom/src/ui/SortBy.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { PartialKeys } from '../types';
+import type { UseSortByProps } from 'react-instantsearch-hooks';
+
+export type SortByProps = Omit<
+  React.HTMLAttributes<HTMLDivElement>,
+  'onChange'
+> &
+  PartialKeys<UseSortByProps, 'items'> &
```

Comment:
```suggestion
 UseSortByProps &
```

(then prettier)

Reply:
This would be `Pick` instead (we don't pass `transformItems` to the UI component) but good catch 👍

Addressed in b3840aa8.